### PR TITLE
[Ready] Cargo pack guild

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
All cargo crates + some basic info on the consoles

### CODE REVIEW

File: citadel-wiki/main/guides/cargo/pack_lists.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+## Crate Locks and Access
+Just like the rest of the station, crates have variable levels of access. Security crates can be opened by security personnel and those with higher access, et cetera. An emag can get into locked crates, however, or you can get resourceful. Buying a crate will almost always tell you what access it needs when your buying it.
+
+## The cargo consoles
+![What the consoles looks like!](https://cdn.discordapp.com/attachments/661373537461993474/661429881372147748/unknown.png "Each console")
+
+Cargo has quite a few consoles, Request Order Console, Ordering Console, Bounty Console, Drop Pod Console.
+The Order consoles can be hacked with a multy tool to unlock some contrand crates that are less then legal. Or can be emaged to give you antag gear, and other goodies! Hacking the consoles also allows you to sell more items, such as skins or feline body parts.
```
Comment (yourdoom9898) [inline citadel-wiki/main/guides/cargo/pack_lists.md]: "multy tool" -> "multitool"
"emaged" -> "emagged"
"antag gear," -> "antag gear"

File: citadel-wiki/main/guides/cargo/pack_lists.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+The Drop Pod or Exspress console can have an upgrade printed at the cargo prolathen to have more packets. But can not be hacked with a multy tool. The Exspress console on the other hand has packets that are only able to be gotten with it.
```
Comment (yourdoom9898) [inline citadel-wiki/main/guides/cargo/pack_lists.md]: "Exspress" -> "Express"
"prolathen" -> "protolathe"
"packets" -> "packs"
". But" -> ", but"
"multy tool" -> "multitool"

File: citadel-wiki/main/guides/cargo/pack_lists.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+<details>
+  <summary>Click to expand!</summary>
+ 
+|Name                                            |Cost|Contents                          |Access Required|Notes                                      |
+|:----------------------- |:--------------------------|:---------------|:------------------------------------------|:--|
+|Bulletproof Armor Crate                         |1250|Bullet Proof Vests - 3            |Armory         |                                           |
+|Bulletproof Helmet Crate                        |1250|Bullet Proof Helmet - 3           |Armory         |                                           |
+|Chemical Implants Crate                         |1700|Remote Chemical implants - 5      |Armory         |Comes with an implanter in the box         |
+|Combat Knives Crate                             |3200|Combat Knifes - 3                 |Armory         |                                           |
+|Combat Shotguns Crate                           |8000|Automatic Combat Shotgun - 3 </br> Bandolier - 3 </br> Box of 12g - 1 </br> Box of Buckshot - 1 |Armory|The shotguns themselfs come pre-loaded, just needs to be chambered|
```
Comment (yourdoom9898) [inline citadel-wiki/main/guides/cargo/pack_lists.md]: "themselfs" -> "themselves"

File: citadel-wiki/main/guides/cargo/pack_lists.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+|DRAGnet gun Crate                               |3250|DRAGnet gun - 2                   |Armory         |                                           |
+|Energy Guns Crate                               |3250|Energy Gun - 3                    |Armory         |                                           |
+|Exile Implants Crate                            |1050|Exile implant - 5                 |Armory         |Comes with an implanter in the box, useless without admins to make an away mission                                                                                            |
+|Mindshield Implants Crate                       |4000|Mindshield Implants - 3           |Armory         |Likely will end up needing 3-4 of these well in convertion based game mode.                                                                                                   |
```
Comment (yourdoom9898) [inline citadel-wiki/main/guides/cargo/pack_lists.md]: "these well in" -> "these in"
"game mode" -> "game modes"

File: citadel-wiki/main/guides/cargo/pack_lists.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+|Tracking Implants Crate                         |1050|Tracking Implant - 4              |Armory         | Comes with an implanter in the box        |
+|Incendiary Weapons Crate                        |1750|Fully made flamer - 1 </br> Plasma tank - 3 </br> Box of Dragon Breath - 1 <br/> Grenade Incendiary - 3                                                                       |Heads |Flamer dosnt come loaded.                  |
```
Comment (yourdoom9898) [inline citadel-wiki/main/guides/cargo/pack_lists.md]: "dosnt" -> "doesn't"

File: citadel-wiki/main/guides/cargo/pack_lists.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+|Personal Miniature Energy Guns                  |3000|Miniature Energy gun - 3          |Armory         |                                           |
+|Reflector Vest Crate                            |2000|Reflector Vest       - 2          |Armory         |Shockingly cheap traiter goal              |
```
Comment (yourdoom9898) [inline citadel-wiki/main/guides/cargo/pack_lists.md]: "traiter" -> "traitor"

File: citadel-wiki/main/guides/cargo/pack_lists.md [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+|Riot Armor Crate                                |1750|Riot Vests - 3                    |Armory         |                                           |
+|Riot Helmets Crate                              |1750|Riot Helmets - 3                  |Armory         |                                           |
+|Riot Shields Crate                              |2200|Riot Shields  - 3                 |Armory         |                                           |
+|Riot Shotgun Crate                              |6500|Riot Shotgun  - 3            </br> Box of rubber shot - 1       </br> Box of Beanbag - 1                                                                                 |Armory|The shotguns themselfs come pre-loaded, just needs to be chambered                                                                                                             |
```
Comment (yourdoom9898) [inline citadel-wiki/main/guides/cargo/pack_lists.md]: "themselfs" -> "themselves"

File: citadel-wiki/main/guides/cargo/pack_lists.md [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+|SWAT Crate                                      |6000|NT Swat Helmets  - 2 </br> SWAT Suits - 2 </br> Combat gloves - 2 </br> SWAT sec Hailers - 2 </br> Combat Belt = 2 |Armory         |The SWAT suit and helmets are space proof  |
+|SWAT tatical tasers Crate                       |7000|SWAT Tasers  - 2                   |Armory        |                                           |
+|Classic WoodStock Shotguns Crate                |3000|4 Round shotgun  - 3               |Armory        |Already loaded needs to be chambered       |
```
Comment (yourdoom9898) [inline citadel-wiki/main/guides/cargo/pack_lists.md]: "loaded needs" -> "loaded but needs"

File: citadel-wiki/main/guides/cargo/pack_lists.md [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+|:----------------------- |:--------------------------|:---------------|:------------------------------------------|:--|
+|Collectable Hats Crate                         |20000|Collectable Hat That can be one of -</br>Chef</br>Paper</br>Tophat</br>Captain</br>Beret</br>Welding</br>Flatcap</br>Pirate</br>Kitty</br>Rabbet Ears</br>Wizard</br>Hardhat</br>HoS</br>HoP</br>Thunderdome</br>SWAT</br>Slime</br>Xenom</br>Ultra Rare Pete's hat!             ||You only get 3 random items in this list                                           |
+|Contraband Crate                                |3000|Contraband Posters</br>Life Weed</br>Cannabis</br>Life Cannabis</br>Zoompill Bottle</br>Happypill Bottle</br>LSDpill Bottle</br>Aranesppill Bottle</br>Stimulantpill Bottle</br>Deck of Syndicate Cards</br>Tacticool Jumpsuit</br>Tactical Jumpsuit</br>Bottle of Absinthe</br>Gun Suppressor</br>Shady Slims Smokes</br>Syndicate Smokes</br>Syndicate Gas Mask</br>Golden Necklace</br>Donksoft Vender Restock Unit</br>Amputation Arcade Game           |Console Needs to be hacked|    You only Get 5 random items in this pack      |
```
Comment (yourdoom9898) [inline citadel-wiki/main/guides/cargo/pack_lists.md]: "Get" -> "get"

File: citadel-wiki/main/guides/cargo/pack_lists.md [hunk 10]
```diff
@@ -0,0 +1,105 @@
...
+|Hilarious Firing Pin Crate                      |5000|Clown Firing Pin - 1              |Console Needs to be hacked|                   |
+|Laser Tag Crate                                 |3500|Red Laser Tag Gun -3</br> Blue Laser Tag Gun -3</br> Red Laser Tag Armor-3</br> Blue Laser Tag Armor-3</br>Red Laser Tag Helmet-3</br>Blue Laser Tag Helmet-3                 |         |                                                                                            |
+|Laser Tag Firing Pins Crate                     |3000|Mindshield Implants - 3           |Armory         |Likely will end up needing 3-4 of these well in convertion based game mode.                                                                                                   |
```
Comment (yourdoom9898) [inline citadel-wiki/main/guides/cargo/pack_lists.md]: From line 84 onwards, it looks like you accidentally copy pasted the armory crates over again

Comment (Trilbyspaceclone) [reply citadel-wiki/main/guides/cargo/pack_lists.md]: Its a wip

**Comment by Trilbyspaceclone** [comment]
Slowly getting it done!

File: citadel-wiki/main/guides/cargo/pack_lists.md [hunk 11]
```diff
@@ -25,9 +25,9 @@ Just like the rest of the station, crates have variable levels of access. Securi
 ![What the consoles looks like!](https://cdn.discordapp.com/attachments/661373537461993474/661429881372147748/unknown.png "Each console")
 
 Cargo has quite a few consoles, Request Order Console, Ordering Console, Bounty Console, Drop Pod Console.
-The Order consoles can be hacked with a multy tool to unlock some contrand crates that are less then legal. Or can be emaged to give you antag gear, and other goodies! Hacking the consoles also allows you to sell more items, such as skins or feline body parts.
+The Order consoles can be hacked with a multitool to unlock some contrand crates that are less then legal. Or can be emagged to give you antag gear and other goodies! Hacking the consoles also allows you to sell more items, such as skins or feline body parts.
```
Comment (yourdoom9898) [inline citadel-wiki/main/guides/cargo/pack_lists.md]: contrand -> contraband

File: citadel-wiki/main/guides/cargo/pack_lists.md [hunk 12]
```diff
@@ -25,9 +25,9 @@ Just like the rest of the station, crates have variable levels of access. Securi
...
 
-The Drop Pod or Exspress console can have an upgrade printed at the cargo prolathen to have more packets. But can not be hacked with a multy tool. The Exspress console on the other hand has packets that are only able to be gotten with it.
+The Drop Pod or Express console can have an upgrade printed at the cargo protolathe to have more packs, but can not be hacked with a multitool. The Exspress console on the other hand has packets that are only able to be gotten with it.
```
Comment (yourdoom9898) [inline citadel-wiki/main/guides/cargo/pack_lists.md]: Missed a second Exspress -> Express here

File: citadel-wiki/main/guides/cargo/pack_lists.md [hunk 13]
```diff
@@ -0,0 +1,138 @@
...
+|Original Costume Crate                          |1750|Snowman Suit - 1</br>Snowman Suit Head-1</br>Chicken Suit-1</br>Chicken Suit Head-1</br>Monkey Suit-1</br>Monkey Mask-1</br>Cardborg Suit-1</br>Cardborg Helmet-1</br>Xeno Suit-1</br>Xeno Suit Head-1</br>Corgi Costume-1</br>Carp Costume-1</br>Bee Costume-1               |         |     |
+|Standard Costume Crate                          |1300|Giggles Von Honkerton-1</br>Clown Shoes-1</br>Clown Wig and Mask-1</br>Clown Suit-1</br>Bike Horn-1</br>Mime's Outfit-1</br>Black Shoes-1</br>White Gloves-1</br>Mime Mask-1</br>French Beret-1</br>Suspenders-1</br>Bottle of Nothing-1</br>Parcel Parceaux-1             |Theatre          |                                           |
+|Toy Crate                                       |5000|Random prizes usually found in arcade machines. </br>Donksoft Vender Restock            |         |Only 5 items in said crate. No Pushies       |
```
Comment (yourdoom9898) [inline citadel-wiki/main/guides/cargo/pack_lists.md]: Pushies -> plushies

**Pull request closed without merging**